clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1439,10 +1439,23 @@ private void processIngestionException() {
             // DaVinci is always a follower.
             subscribePartition(pubSubTopicPartition, false);
           }
+        } else if (isCurrentVersion.getAsBoolean() && resetErrorReplicaEnabled && !isDaVinciClient) {
```

Comment:
Is this the right place to mark replicas ERROR? IIUC we may not call `processIngestionException` if `lastStoreIngestionException` is set.